DDD Workshop.ipynb

```diff
@@ -0,0 +1,547 @@
+    "\n",
+    "This phenomenon is very possible in the transfer of information at any stage from data sources to scientists to reporters to consumers, again intentional or unintentional. We see this taken to a comedic level with XKCD's exploration of jelly beans:\n",
+    "\n",
+    "![](https://i.imgur.com/k2dDf9e.png)\n",
+    "\n",
+    "\n",
+    "What you should take away here is that data manipulated is not meaningful, and I argue it is not even information, as it does not represent knowledge in any real way. Let's not be too cynical now, there _is_ meaningful information in the world, and we want to separate it from this darker part of the data sphere.\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "\n",
+    "## Meaningful Information\n",
+    "\n",
+    "Here we have information, not misleading or malicious, and want to determine what to do with it. So then, what makes your information worth producing and using?\n",
+    "\n",
+    "#### \"Useful\" Information\n",
+    "\n",
+    "When we say \"useful\", what we want is SMART. SMART is an acronym that is generally used for goal-setting, where it stands for:\n",
+    "\n",
+    "- Specific: this information should be scoped to answer the question(s) you have, as information too broad risks being less applicable to the situation\n",
+    "- Measurable: an excellent metric for what to do with your data, and the subject of the next part\n",
+    "- Attainable: similar to specificity, information too specific might be too difficult to build a solution for, or not worth the development time\n",
+    "- Relevant: if your information is not about your subject, it won't be a good basis for solutions\n",
+    "- Timely: while data being too old doesn't mean that it's wrong, it makes the correlation to today weaker and harder to build for. Likewise, a projection too far in the future might not be appropriate for your current project or plans.\n",
+    "\n",
+    "\n",
+    "<!-- cite https://www.mindtools.com/pages/article/smart-goals.htm -->\n",
+    "\n",
+    "These are the things that you want in goal setting for sure, but we also want them in information, and it's no coincidence (p > 0.05). We want information that supports the goals you have when using it. Otherwise, information is as good as trivia, which is not bad necessarily, but useless by definition.\n",
```

Comment:
Would pick a word that is more specific than "things." Maybe reword to something like this: All of the above are important to keep in mind when goal setting. We also want them in information, and it's no coincidence...